src/agent/border_agent.cpp

```diff
@@ -145,8 +152,15 @@ otbrError BorderAgent::Start(void)
 
 void BorderAgent::Stop(void)
 {
+#if OTBR_ENABLE_ADVERTISING_PROXY
+    mAdvertisingProxy.Stop();
+#endif
+
```

Comment:
duplicate with below Line#161~163

```suggestion
```

Reply:
Thanks, didn't catch this when rebasing the PR.